dev/devenv-readme.md

```diff
@@ -32,3 +34,9 @@ Unfortunately, this project will sometimes need to depend on an interim/snapshot
     $ mvn clean install -DskipITs=true -DskipTests=true
 
 Once the build is done, the HAPI FHIR artifacts will be placed into your user's local Maven repository (`~/.m2/repository`), available for use by this project or others.
+
+### AWS Credentials
+
+Some of this project's integration tests require AWS credentials. These credentials are used to stand up and tear down AWS resources used in the tests. For example, `DataSetMonitorIT` uses the AWS API to create and manipulate the S3 buckets that it uses during the tests, removing them at the end of each test case. This test code uses the AWS Java API's [DefaultAWSCredentialsProviderChain](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html) class, which will supports multiple different mechanisms for retrieving the credentials to use. Credentials must be supplied via at least one of those mechanisms, or the ITs will fail.
+
+(The project's `Jenkinsfile` uses the environment variable mechanism, which [the project's main Jenkins server](http://builds.hhsdevcloud.us/) has been configured to support.)
```

Comment:
I think I'll pass, as folks actually really should crack that open and look at the list of chained providers. Per conversation with Mike this morning, we'll likely end up using env vars during local dev & test, and the instance profile in the HealthAPT dev/test/prod systems.